cmd/root.go

```diff
@@ -51,11 +51,12 @@ oh-my-zsh alias is disabled (e.g. $ \gup update).
 }
 
 // Execute run gup process.
-func Execute() {
+func Execute() (err error) {
 	assets.DeployIconIfNeeded()
 	rootCmd := newRootCmd()
 
-	if err := rootCmd.Execute(); err != nil {
+	if err = rootCmd.Execute(); err != nil {
 		print.Err(err)
 	}
+	return
```

Comment:
```suggestion
	return rootCmd.Execute()
```

And, print error message at main.go
```go
	if err := cmd.Execute(); err != nil {
		print.Err(err)
	}
```

Reply:
Previously, Execute printed the errors it sees, and ignored them otherwise, so this PR did not really change that. But I agree it's better to handle printing on higher level, done in 4c293f54d4ca8a08bb69dfa7dbc0781123c386a6.